styles.js

```diff
@@ -5,39 +5,86 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
+    font-family: sofia-pro, sans-serif;
+    color: var(--text-color-dark);
+    font-size: var(--font-size-sm);
+    line-height: 1.5;
+    margin: 0;
+    padding: 0;
+    border: none;
   }
 
   :root {
+    /* color */
     --light-bg-color: #F7F9FB;
+    --white-bg-color: #ffffff;
     --dark-grey-color: #d4d4d4;
-    --accent-color: #e9ff70;
     --text-color-dark: #141414;
+    --accent-color: #e9ff70;
     --friendly-red-color: #F44336;
     --friendly-green-color: #4CAF50;
```

Comment:
typically you would not choose "light" "dark" or colors like "red" as the variable names. For your case its ok since you don't have any other themes, but as soon as you want to implement a dark mode, the "dark-grey-color" might not work anymore because the same element would need a "light-grey-color" instead to be visible on the dark background. 
Therefore, a common apporoach is to call the colors after their role in the design: "background" "highlight" "accent" "danger". The new Material v3 Design System established the naming pattern "surface", "surface-on" "elevated" and "elevated-on" with the "on" color being used for text, icons, everything that is displayed on the element. Elevated means elements that should appear more closely to the user like cards, navigation bars etc. 
Feel free to ignore this, since it is not relevant in your case, but its interesting none the less.

Reply:
Thank you very much for the tip!